### PR TITLE
Fix the wheel CI

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by wizmer** [comment]
@mgeplf `manylinux2010` does not have `cmake >= 3` which is required for morphio.
I can build cmake from sources and use manylinux2010 if you want.
My personal opinion is that 2014 is fine, all our machine at BBP are 2014 compliant. If someone comes here and complain, we can reconsider but chances are that it will never happen.

File: .github/workflows/publish-sdist-wheels.yml [hunk 1]
```diff
@@ -45,7 +45,11 @@ jobs:
       - name: Build wheels Mac OS
         if: runner.os == 'macOS'
         env:
-          CIBW_BEFORE_BUILD: 'brew update; brew unlink gcc@8 gcc@9; brew install cmake hdf5'
+          CIBW_BEFORE_BUILD: |
```
Comment (mgeplf) [inline .github/workflows/publish-sdist-wheels.yml:48]: is there a behavior change here?

Comment (wizmer) [reply .github/workflows/publish-sdist-wheels.yml:48]: No, I think the only change was in brew itself.

Comment (mgeplf) [reply .github/workflows/publish-sdist-wheels.yml:48]: k, thanks.

**Pull request merged**